runtime/env.cpp

```diff
@@ -0,0 +1,40 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2023 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <cstdlib>
+#include <unistd.h>
+
+#include "runtime/env.h"
+
+static bool valid_environment_name(const char *name, const char *end) noexcept {
+  for (const char *s = name; s < end; ++s) {
+    if (*s == ' ' || *s == '.' || *s == '[') {
+      return false;
+    }
+  }
+  return true;
```

Comment:
Wouldn't 
```c++
return std::find_if(name, end, [](char s) {
  return vk::any_of_equal(s, ' ', '.', '[');
} == end;
```
be more compact and idiomatic?

Reply:
true, it's done